htdocs/product/class/product.class.php

```diff
@@ -1677,15 +1677,15 @@ function updatePrice($newprice, $newpricebase, $user, $newvat='',$newminprice=''
 			return -1;
 		}
 
-		if ($newprice!='' || $newprice==0)
+		if (is_numeric($newprice) || $newprice == '0')
```

Comment:
If $newprice is '5,5' (french format) or '5X5' with X is decimal separator of another language, it is not yet a decimal.
It will ba later when converted with price2num.
For this reason, we can't use is_numeric here. To avoid warning, may be this may work

if (((string) $newprice) != '')

Reply:
@eldy 
same problem with (string), but it's ok with default value 0 for $newminprice parameter https://github.com/Dolibarr/dolibarr/pull/8178